DOCUMENTATION.md

```diff
@@ -1,4 +1,5 @@
 # Documentation
+
 Welcome to our documentation guide. Here are some useful tips you need to know!
 
 # Table of Contents
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project